pkg/filters/args.go

```diff
@@ -19,6 +20,55 @@ func NewArgFilter() *ArgFilter {
 	}
 }
 
+func matchFilter(filters []string, argValStr string) bool {
+	for _, f := range filters {
+		prefixCheck := f[len(f)-1] == '*'
+		if prefixCheck {
+			f = f[0 : len(f)-1]
+		}
+		suffixCheck := f[0] == '*'
+		if suffixCheck {
+			f = f[1:]
+		}
+		if argValStr == f ||
+			(prefixCheck && !suffixCheck && strings.HasPrefix(argValStr, f)) ||
+			(suffixCheck && !prefixCheck && strings.HasSuffix(argValStr, f)) ||
+			(prefixCheck && suffixCheck && strings.Contains(argValStr, f)) {
+			return true
+		}
+	}
+	return false
+}
+
+func (filter *ArgFilter) Filter(eventID events.ID, args []trace.Argument) bool {
+	if filter.Enabled {
```

Comment:
to avoid extra indentation, you can use:
```
if !filter.Enabled {
    return true
}
```

Reply:
Sure, that's how I did it in #1750